pyproject.toml

```diff
@@ -53,4 +55,10 @@ extend_skip = [
   "tools",
 ]
 skip_gitignore = "True"
+src_paths = [
+  "docs",
+  "lib",
+  "noxfile.py",
+  "setup.py",
+]
```

Comment:
`tools` dir as well?

Reply:
Some of the tools are auto generated code, so we shouldn't be touching those.